backend/layers/persistence/orm.py

```diff
@@ -10,71 +10,61 @@
 
 
 @mapper_registry.mapped
-class Collection:
+class CollectionTable:
 
-    __table__ = Table(
-        "Collection",
-        mapper_registry.metadata,
-        Column("id", UUID(as_uuid=True), primary_key=True),
-        Column("version_id", UUID(as_uuid=True)),
-        Column("originally_published_at", DateTime),
-        Column("tombstoned", BOOLEAN),
-    )
+    __tablename__ = "Collection"
+
+    id = Column(UUID(as_uuid=True), primary_key=True)
+    version_id = Column(UUID(as_uuid=True))
+    originally_published_at = Column(DateTime)
+    tombstone = Column(BOOLEAN)
 
 
 @mapper_registry.mapped
-class CollectionVersion:
-
-    __table__ = Table(
-        "CollectionVersion",
-        mapper_registry.metadata,
-        Column("version_id", UUID(as_uuid=True), primary_key=True),
-        Column("collection_id", UUID(as_uuid=True)),
-        Column("metadata", JSON),
-        Column("owner", String),
-        Column("curator_name", String),
-        Column("publisher_metadata", JSON),
-        Column("published_at", DateTime),
-        Column("created_at", DateTime),
-        Column("datasets", ARRAY(UUID(as_uuid=True))),
-    )
+class CollectionVersionTable:
+
+    __tablename__ = "CollectionVersion"
+
+    id = Column(UUID(as_uuid=True), primary_key=True)
+    collection_id = Column(UUID(as_uuid=True))
+    collection_metadata = Column(JSON)
```

Comment:
In order to be able to use `metadata` you'll need to switch to [declarative-mapping](https://docs.sqlalchemy.org/en/14/orm/mapping_styles.html#declarative-mapping)
```python
from sqlalchemy import Column, Integer, String, ForeignKey
from sqlalchemy.orm import declarative_base

# declarative base class
Base = declarative_base()

# an example mapping using the base
class User(Base):
    __tablename__ = "user"

    id = Column(Integer, primary_key=True)
    name = Column(String)
    fullname = Column(String)
    nickname = Column(String)
```

or you can try mapping the column like this: 
```suggestion
    collection_metadata = Column("metadata", JSON)
```

Reply:
I agree it would be preferable to keep `metadata` and use Trent's solution.

Reply:
I'm getting a similar error when I try this: sqlalchemy.exc.InvalidRequestError: Attribute name 'metadata' is reserved for the MetaData instance when using a declarative base class.

Reply:
The second solution will work for sure, but then we will have different naming. Not the end of the world IMO.

Reply:
FWIW we'd still need to do a migration to fix the id naming (i.e. for DatasetVersions, renaming dataset_version_id -> id, etc.) so at that point I think I'd rather migrate the metadata names as well and keep internal consistency with the naming. 

Alternatively, with imperative mapping (the 'Table' object mapping we're currently doing in main) we can keep the metadata names. Still requires migration for the Id rename though, which I do think is worth fixing.